docs/source/getting-started/preparing-to-install.adoc

```diff
@@ -0,0 +1,61 @@
+== Overview
+
+The following section describes how to install {project} and the required dependencies.
+
+The {project} setup procedure should be run as a regular user with permission to launch virtual machines.
+In the procedure, you will see how to assign that permission, along with ways to configure your hypervisor and command shell to start and effectively interact with {project}.
+
+[[install-prerequisites]]
+== Prerequisites
+
+{project} requires a hypervisor to start the virtual machine on which the OpenShift cluster is provisioned.
+Verify that the hypervisor of your choice is installed and enabled on your system before you set up {project}.
+Once the hypervisor is up and running, additional setup is required for {project} to work with that hypervisor.
+
+Depending on your host operating system, you have the choice of the following recommended native hypervisors:
+
+macOS::
+- link:https://github.com/mist64/xhyve[xhyve]
+
+Linux::
+- link:https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine[KVM]
+
+Windows::
+- link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V]
++
+[NOTE]
+====
+To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], you also link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] using the Hyper-V Manager and set the environment variable `HYPERV_VIRTUAL_SWITCH` to this virtual switch.
+For specific configuration steps see the xref:../getting-started/setting-up-virtualization-environment.adoc#[Setting Up the Virtualization Environment] section.
+====
+
+link:https://www.virtualbox.org/wiki/Downloads[VirtualBox] may also be used for all supported operating systems instead of the recommended native hypervisor.
+
+[NOTE]
+====
+VirtualBox 5.1.12 or later is recommended on Windows to avoid the issue xref:../troubleshooting/troubleshooting-driver-plugins.adoc#machine-doesnt-exist[Error: machine does not exist].
+If you encounter issues related to the hypervisor, see the xref:../troubleshooting/troubleshooting-driver-plugins.adoc#troubleshooting-driver-virtualbox[Troubleshooting Driver Plug-ins] section.
+====
+
+Refer to the documentation for each hypervisor to determine the hardware and operating system versions needed to run that hypervisor.
+
+== Understanding {project} Installation
+These are the basic steps for setting up {project} on your personal system:
+
+. xref:../getting-started/setting-up-virtualization-environment.adoc#[Set up your virtualization environment]
+. Download {project} software for your operating system from the link:{download-url}[{download-text}] page
+. xref:../getting-started/installing.adoc#[Install {project}]
+. xref:../getting-started/quickstart.adoc#starting-minishift[Start {project}]
+. Configure {project} so you can use it efficiently
```

Comment:
No link for this? I thought you you want to redirect user to the appropriate section.

Reply:
Oversight on my part, this is intended to link to **_using/basic_usage.adoc#runtime-options_**.